be/src/runtime/tablet_writer_mgr.cpp

```diff
@@ -311,6 +313,44 @@ Status TabletWriterMgr::cancel(const PTabletWriterCancelRequest& params) {
     {
         std::lock_guard<std::mutex> l(_lock);
         _tablets_channels.erase(key);
+        _key_time_map.erase(key);
+    }
+    return Status::OK;
+}
+
+Status TabletWriterMgr::start_bg_worker() {
+    _tablets_channel_clean_thread = std::thread([this] {
+                _tablets_channel_clean_thread_callback(nullptr);
```

Comment:
```suggestion
        uint32_t interval = 60;
        while (true) {
            _start_tablets_channel_clean();
            sleep(interval);
        }
```
I think 600 is a bit long

Reply:
OK